admin-dev/themes/default/js/bundle/product/form.js

```diff
@@ -2052,10 +2052,12 @@ var seo = (function() {
       case '302-category':
         productRedirect.find('label').html(redirectTypeElem.attr('data-labelcategory'));
         productRedirect.find('input').attr('placeholder', redirectTypeElem.attr('data-placeholdercategory'));
+        productRedirect.find('.typeahead-hint').html(redirectTypeElem.attr('data-hintcategory'));
```

Comment:
Prefer `.text()` if you do not have html content to store here.

Reply:
done!